server/store.go

```diff
@@ -205,3 +206,66 @@ func (p *Plugin) removeSuperUserToken() error {
 
 	return nil
 }
+
+func (p *Plugin) storeUserPreference(userID, value string) error {
+	encoded, err := json.Marshal(value)
+	if err != nil {
+		return err
+	}
```

Comment:
I do not think we need to encode it here. The logic is already handled inside client.KV.Set